.github/workflows/operator-integration-test.yml

```diff
@@ -71,47 +72,76 @@ jobs:
 
       - name: Test for defined instrumentation resources for Java
         run: |
           kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
-          sleep 5
           kubectl rollout restart deployment nginx
-          sleep 5
+          sleep 3
+          kubectl wait --for=condition=Ready pod --all -n default
           kubectl get pods -A
           kubectl describe pods -n default
           cd integration-tests/manifests/cmd
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json     
           kubectl delete instrumentation sample-instrumentation
+          sleep 2
+          kubectl wait --for=condition=Ready pod --all -n default
+
 
       - name: Test for default instrumentation resources for python
         run: |
           kubectl apply -f integration-tests/python/sample-deployment-python.yaml
-          sleep 5
           kubectl rollout restart deployment nginx
-          sleep 5
+          sleep 3
+          kubectl wait --for=condition=Ready pod --all -n default
           kubectl get pods -A 
           kubectl describe pods -n default
           go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/python/default_instrumentation_python_env_variables.json
 
-
       - name: Test for defined instrumentation resources for python
         run: |
           kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
-          sleep 5
           kubectl rollout restart deployment nginx
-          sleep 5
+          sleep 3
+          kubectl wait --for=condition=Ready pod --all -n default
           kubectl get pods -A
           kubectl describe pods -n default
           cd integration-tests/manifests/cmd
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
           kubectl delete instrumentation sample-instrumentation
+          kubectl wait --for=condition=Ready pod --all -n default
+
+      - name: Test for default instrumentation resources for python and java
+        run: |
+          kubectl apply -f integration-tests/python-java/sample-deployment-python-java.yaml
+          sleep 3
+          kubectl wait --for=condition=Ready pod --all -n default
+          kubectl get pods -A 
+          kubectl describe pods -n default
+          kubectl describe pods -n amazon-cloudwatch
+          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/python-java/default_instrumentation_python-java_env_variables.json
+
 
 
+      - name: Test for defined instrumentation resources for python and java
+        run: |
+          kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
+          kubectl delete pods --all -n default
+          sleep 3
+          kubectl wait --for=condition=Ready pod --all -n default
+          kubectl rollout restart deployment nginx
```

Comment:
why are we deleting pods and then doing a rollout restart anyway? We should probably choose 1.

Maybe just do the kubectl rollout and then follow up with kubectl wait